core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -1302,6 +1317,34 @@ wasm_cluster_set_exception(WASMExecEnv *exec_env, const char *exception)
     data.skip = NULL;
     data.exception = exception;
 
+#if WASM_ENABLE_DUMP_CALL_STACK != 0
+    if (has_exception) {
+        /* Save the stack trace of the crashed thread into the cluster */
+        WASMModuleInstance *module_inst =
+            (WASMModuleInstance *)get_module_inst(exec_env);
+
+#if WASM_ENABLE_INTERP != 0
+        if (module_inst->module_type == Wasm_Module_Bytecode
+            && wasm_interp_create_call_stack(exec_env)) {
+            wasm_interp_dump_call_stack(exec_env, false, NULL, 0);
```

Comment:
What's the point of the call? We ignore the return value, we also pass a NULL buffer and don't print the stack.

Reply:
The `print=true` is a typo, it should be `false` there as well, thanks for noticing. The calls to `wasm_interp_create_call_stack()` and `wasm_interp_dump_call_stack()` are just to dump the stack trace so that it can be copied into the thread manager.

Reply:
I understand why `wasm_interp_create_call_stack` is required but I don't understand what's the purpose of calling `wasm_interp_create_call_stack` given that you pass NULL buffer, disable printing and ignoring return value?

Reply:
True, I misread the function body, I thought it was populating some of the frame fields but it does not. So, yeah, that function call can be removed.